### PR TITLE
Rename project to pymc-marketing

### LINKED ISSUE #101: Rename `pymmmc` directory to ... ?
... just `marketing`

### LINKED ISSUE #112: Put `distributions.py` to `clv` submodule
They are all CLV-specific (for now)

### PR DESCRIPTION
**Pull request by ricardoV94**
Closes #101 
Closes #112
Clases #79

Regarding the import name, I think in most cases we can show:
```python
from pymc_marketing import mmm
from pymc_marketing import clv
```

### CODE REVIEW

**Comment by ricardoV94** [comment]
Commits have semantic meaning, so feel free to see one at a time if it helps with review

File: docs/source/conf.py [hunk 1]
```diff
@@ -110,7 +110,7 @@
     "navbar_start": ["navbar-logo", "navbar-name"],
     "navbar_end": ["theme-switcher"],
     "footer_items": ["copyright", "sphinx-version", "theme-version", "footer-links"],
-    "github_url": "https://github.com/pymc-labs/pymmmc",
+    "github_url": "https://github.com/pymc-labs/pymc_marketing",
```
Comment (twiecki) [inline docs/source/conf.py:113]:
```suggestion
    "github_url": "https://github.com/pymc-labs/pymc-marketing",
```

File: docs/source/conf.py [hunk 2]
```diff
@@ -139,7 +139,7 @@
 }
 html_context = {
     "github_user": "pymc-labs",
-    "github_repo": "pymmmc",
+    "github_repo": "pymc_marketing",
```
Comment (twiecki) [inline docs/source/conf.py:142]:
```suggestion
    "github_repo": "pymc-marketing",
```

File: docs/source/conf.py (hunk 1 above)
Comment (twiecki) [reply docs/source/conf.py:113]: Or should we rename the repo?

Comment (ricardoV94) [reply docs/source/conf.py:113]: I this makes sense

File: setup.py [hunk 3]
```diff
@@ -33,7 +33,7 @@ def get_version():
 
 
 setup(
-    name="pymmmc",
+    name="pymc_marketing",
```
Comment (twiecki) [inline setup.py:36]: This is the name it'll get on pypi, no? So I guess same question.

Comment (ricardoV94) [reply setup.py:36]: In pymc-experimental it seems to have the underscore? https://github.com/pymc-devs/pymc-experimental/blob/5a01d4fc5d58fc1abb52f13fcb5cd4e8b32292ba/setup.py#L22

File: docs/source/api/index.md [hunk 4]
```diff
@@ -2,24 +2,19 @@
 
 ## Top level objects
 ```{eval-rst}
-.. currentmodule:: pymmmc
+.. currentmodule:: pymc-marketing
```
Comment (OriolAbril) [inline docs/source/api/index.md:5]: This needs to be the string used to import the package/module. Now sphinx can't import anything and the api docs are empty: https://pymc-marketing--116.org.readthedocs.build/en/116/api/index.html
I also think this section is no longer needed as there are no longer top level objects right? Only the two modules which are listed below

File: docs/source/api/index.md [hunk 5]
```diff
@@ -2,24 +2,19 @@
...
 
-
-  ContNonContract
-  ContContract
 ```
 
 ## Modules
 ```{eval-rst}
-.. currentmodule:: pymmmc
+.. currentmodule:: pymc-marketing
```
Comment (OriolAbril) [inline docs/source/api/index.md:13]: Same comment here. Here I add the suggestion as this section definitely needs to be there

```suggestion
.. currentmodule:: pymc_marketing
```

File: setup.py (hunk 3 above)
Comment (ricardoV94) [reply setup.py:36]: Yeah, that looks like solid evidence to me :D

**Review by OriolAbril — CHANGES_REQUESTED** [review]

**Review by OriolAbril — APPROVED** [review]
Noticed a last comment that cpuld go either way, other than that approving on my end

File: docs/source/conf.py [hunk 6]
```diff
@@ -2,12 +2,12 @@
...
 import os
 
-import pymmmc  # isort:skip
+import pymc_marketing  # isort:skip
 
 # -- General configuration ------------------------------------------------
 
 # General information about the project.
-project = "pymmmc"
+project = "pymc_marketing"
```
[the remark this answers is not in the document]
Comment (ricardoV94) [reply docs/source/conf.py:10]: I think the install

**Pull request merged**